src/menu-slider/ia-menu-slider.ts

```diff
@@ -150,9 +150,9 @@ export class IaMenuSlider extends LitElement {
             @menuTypeSelected=${this.setSelectedMenu}
           >
             ${this.renderMenuHeader}
-            <section>
+            <main>
```

Comment:
question: Should `<main>` be reserved for the theater section?

source: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main
> The content of a <main> element should be unique to the document. Content that is repeated across a set of documents or document sections such as sidebars, navigation links, copyright information, site logos, and search forms shouldn't be included unless the search form is the main function of the page.

Reply:
TIL! Cool, will fix :+1: